hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/meta/bni/Dispatch.java

```diff
@@ -42,6 +42,14 @@ public interface Dispatch {
     // --- {@code contractCallResult} has the given sender contract number; and whose result is derived
     // --- from the returned state via a given {@link ResultTranslator}
 
+    /**
+     * Returns the all the bytes of entropy available in this scope.
```

Comment:
_All_ bytes of entropy?  What are we usually talking about here? (Does there need to be a limit, like, up to 32 bytes of entropy?  (Can you run out in the scope if you call it too often?)

(As an aside, I just love the way we use the term "entropy" in computer science.  Probably gives the physicists good reason to sneer at us "science" pretenders, but who cares?)

Reply:
Haha. 😝 Linked [HIP-351](https://hips.hedera.com/hip/hip-351) to indicate what "entropy" means here.